Examples/TodoApp/TodoApp/src/tasks/domain/TasksList.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+struct TasksList {
```

Comment:
An `enum` would be more idiomatic here

Reply:
Is the desire here to prevent `TasksList` from being instantiated, or is there a different reason for using an `enum` instead?

Reply:
To prevent it from being instantiated, yes

Reply:
I would argue that not using a fake namespace would be more idiomatic. Namespaces are for avoiding name conflicts, not creating taxonomies.